pkg/env/readers.go

```diff
@@ -126,3 +138,16 @@ func FakeTestConfig(fName string, ptr interface{}) func() {
 		overrides.delete(fName)
 	}
 }
+
+// FakeTestConfigHandler allows you to fake the test config with a specific value.
+// and returns an error if a config with the same name exists already
```

Comment:
Period on wrong line. I would also expand very slightly to note how to handle errors:
```suggestion
// FakeTestConfigHandler allows you to fake the test config with a specific value
// and returns an error if a config with the same name exists already. If callers get an error,
// they should switch to running tests in serial.
```